lib/mandriller.rb

```diff
@@ -1,6 +1,6 @@
+require 'action_mailer'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.